examples/file-transfer/src/__tests__/acceptance/test-helper.ts

```diff
@@ -40,6 +40,6 @@ export interface AppWithClient {
 
 const SANDBOX = path.resolve(__dirname, '../../../.sandbox');
 export function getSandbox() {
-  const sandbox = new TestSandbox(SANDBOX);
+  const sandbox = new TestSandbox(SANDBOX, '');
```

Comment:
What is the reasoning for using an empty subdir string, i.e. using the same sandbox dir from all (child) processes? Did you mean to use the default auto-generated subdir instead?

```suggestion
  const sandbox = new TestSandbox(SANDBOX);
```